lesson_9/lesson_9_zoldi_razvan/lesson_9_zoldi_razvan.py

```diff
@@ -0,0 +1,94 @@
+# Build two classes of your choice that can model a real-life example. The class needs to meet the following
+# requirements:
+#
+# - at least 5 attributes each
+# - at least 2 methods each
+# - one class to inherit from another
+#
+# As a demonstration create at least 5 instances of one class (preferably the child class) and call all the methods
+# it holds
+#
+# Ex: You can have one class (Country) that has general attributes about countries such as area, neighbours,
+# cities etc and methods related to those attributes. The second class can be a specific country (Romania) that has
+# more specific attributes such as attractions, universities etc.
+
+# add a comparison country
+ref_country = ['Austria', 'Europe', 'Central Europe', '83879', '+43', 12, 8.85, 2.75, 34, 60]
+
+# 83879 square kilometers
+# 12 major cities
+# 8.85 total population
+# 2.75 population in major cities
+# 34 universities
+# 60 touristic attractions
+
+
+class Country(object):
+    def __init__(self, name, continent, position, area, code, major_cities,
+                 population_country, populations_major_cities):
+        self.name = name
+        self.continent = continent
+        self.position = position
+        self.area = area
+        self.code = code
+        self.major_cities = major_cities
+        self.population_country = population_country
+        self.populations_major_cities = populations_major_cities
+
+    def location(self):
+        if self.continent is ref_country[1] and self.position is ref_country[2]:
+            return f'Both, {ref_country[0]} and {self.name}, are on same continent ({self.continent}) ' \
+                   f'and same area: {self.position}'
+        elif self.continent is ref_country[1] and self.position is not ref_country[2]:
+            return f'Both, {ref_country[0]} and {self.name}, are on same continent ({self.continent}) ' \
+                   f'but on different areas: \n{ref_country[0]} is on {ref_country[2]} area ' \
+                   f'and {self.name} is on {self.position} area'
+        elif self.continent is not ref_country[1]:
+            return f' {ref_country[0]}  and {self.name} are on different continents'
+
+    def area_comparison(self):
+        if int(self.area) == int(ref_country[3]):
```

Comment:
You can avoid the int conversion if you initialize the instances directly with int values